project_template/_base/project_name/apps/common/tests/test_utils_enumeration.py

```diff
@@ -0,0 +1,165 @@
+import pytest
```

Comment:
I'm not agree with you on this point.

All code written by us is custom and could be changed during project's lifetime (instead of Django or other 3rd parties).

And my opinion is the following: everything that could be covered with tests should be covered with tests. For sure there are could be en exceptions like admin class customizations.